umbraco-cloud/set-up/media/connect-programatically-to-azure-storage.md

```diff
@@ -0,0 +1,165 @@
+---
+description: "Connect to Cloud environment's Azure Blob Storage programmatically."
+---
+
+# Connect to Azure Storage Explorer programmatically
+
+This article provides the necessary steps needed to programmatically connect to your Azure Blob Storage containers and upload the needed files, without being publicly available. One will need to have access to the credentials of the Blob Storage to authenticate and find the files created programmatically in the Azure Blob Storage.
```

Comment:
provide feedback from Tony
```suggestion
This article provides the steps needed to programmatically connect to your Umbraco Cloud Environment's Azure Blob Storage containers, in case you want to persist files programmatically. One will need to have access to the credentials of the Blob Storage to authenticate and find the files created programmatically in the Azure Blob Storage.
```